__tests__/models/partial_tx.test.js

```diff
@@ -178,7 +178,7 @@ describe('PartialTx.isComplete', () => {
       new ProposalOutput(1, Buffer.from([])),
       new ProposalOutput(1, Buffer.from([]), { token: '2' }),
       // Add authority output for token 2
-      new ProposalOutput(1, Buffer.from([]), { token: '2', tokenData: TOKEN_AUTHORITY_MASK | 1 }),
+      new ProposalOutput(1, Buffer.from([]), { token: '2', authorities: 1 }), // mint
```

Comment:
A suggestion for reader clarity on these tests:
instead of using `authorities` with values 1, 2 or 3, I would suggest using the mask constants:
```js
  { authorities: TOKEN_MINT_MASK }, // value 1
  { authorities: TOKEN_MELT_MASK }, // value 2
  { authorities: TOKEN_MINT_MASK | TOKEN_MELT_MASK }, // value 3
```
That way it's clearer what these authorities numeric values mean for the readers.

Reply:
Done